lib/hanami/cli/generators/gem/app/gitignore.erb

```diff
@@ -1,2 +1,6 @@
 .env
 log/*
+<%- if Hanami.bundled?("hanami-assets") -%>
+public/
```

Comment:
Question: should we be checking in a public/.keep to make sure this directory exists in the first place?